generate.sh

```
@@ -4,6 +4,8 @@ set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "$SCRIPT_DIR/lib/common.sh"
+BACKUP_ROOT="${MY_AI_TOOLS_BACKUP_DIR:-$HOME/.ai-tools/backups}"
+BACKUP_DIR="$BACKUP_ROOT/generate-$(date +%Y%m%d-%H%M%S)"
```

Shellcheck (0.11.0)

[warning] BACKUP_DIR appears unused. Verify use (or export if used externally).

(SC2034)